lib/node_modules/@stdlib/math/base/ops/cneg/lib/index.js

```diff
@@ -19,32 +19,51 @@
 'use strict';
 
 /**
-* Negate a complex number.
+* Negate a double-precision complex floating-point number.
 *
 * @module @stdlib/math/base/ops/cneg
 *
 * @example
+* var Complex128 = require( '@stdlib/complex/float64' );
+* var real = require( '@stdlib/complex/real' );
+* var imag = require( '@stdlib/complex/imag' );
 * var cneg = require( '@stdlib/math/base/ops/cneg' );
 *
-* var v = cneg( -4.2, 5.5 );
-* // returns [ 4.2, -5.5 ]
+* var z1 = new Complex128( -4.2, 5.5 );
+* // returns <Complex128>
 *
-* v = cneg( 0.0, 0.0 );
-* // returns [ -0.0, -0.0 ]
+* var out = cneg( z1 );
+* // returns <Complex128>
 *
-* v = cneg( NaN, NaN );
-* // returns [ NaN, NaN ]
+* var re = real( out );
+* // returns 4.2
 *
-* @example
-* var cneg = require( '@stdlib/math/base/ops/cneg' );
+* var im = imag( out );
+* // returns -5.5
+*
+* var z2 = new Complex128( 0.0, 0.0 );
+* // returns <Complex128>
+*
+* var out = cneg( z2 );
```

Comment:
Similar to a previous comment, we should either split into separate examples or remove duplicate `var` declarations.